apps/showcase/src/app/modules/rxidb-showcase/components/rxidb-auto-increment/rxidb-auto-increment.component.ts

```diff
@@ -0,0 +1,174 @@
+import { Component, Inject, OnDestroy, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import {
+  addItem,
+  connectIndexedDb,
+  deleteIndexedDb,
+  deleteItem,
+  entries,
+  getObjectStore,
+  keys,
+  read,
+} from '@this-dot/rxidb';
+import { isTruthy } from '@this-dot/utils';
+import {
+  BehaviorSubject,
+  combineLatest,
+  concatMap,
+  EMPTY,
+  from,
+  map,
+  Observable,
+  Subject,
+  switchMap,
+  takeLast,
+  tap,
+} from 'rxjs';
+import { filter, take, takeUntil } from 'rxjs/operators';
+
+const DATABASE_NAME = 'AUTO_INCREMENT';
+
+@Component({
+  selector: 'this-dot-rxidb-auto-increment',
+  templateUrl: './rxidb-auto-increment.component.html',
+  styleUrls: ['./rxidb-auto-increment.component.scss'],
+  providers: [
+    {
+      provide: 'STORE',
+      useValue: connectIndexedDb(DATABASE_NAME).pipe(
+        getObjectStore('store', { autoIncrement: true })
+      ),
+    },
+  ],
+})
+export class RxidbAutoIncrementComponent implements OnInit, OnDestroy {
+  private destroy$ = new Subject<void>();
+  private isLoadingSubject = new BehaviorSubject(false);
+  private readonly keys$ = this.store$.pipe(keys());
```

Comment:
Might just be my machine as the example on Stackblitz seems to be working fine here, but I'm getting incompatible type errors for both `keys()` and `entries()`.

For keys:
Argument of type '(s$: Observable<IDBObjectStore>) => Observable<IDBValidKey[]>' is not assignable to parameter of type 'OperatorFunction<IDBObjectStore, IDBValidKey[]>'.
  Types of parameters 's$' and 'source' are incompatible.

For entries:
  Argument of type '(s$: Observable<IDBObjectStore>) => Observable<undefined>' is not assignable to parameter of type 'OperatorFunction<IDBObjectStore, undefined>'.
  Types of parameters 's$' and 'source' are incompatible.

Reply:
I don't get this type error. Neither locally nor in stackblitz. 👀